locale/de/locale.cfg

```diff
@@ -0,0 +1,31 @@
+[item-name]
+bni_battery-pack=Geladenes Akku Pack
```

Comment:
Akku ist maskulin according to the Duden. It should be geladener. Same with the next 3 lines.